api/sirsidynix_authentication_provider.py

```diff
@@ -179,32 +162,24 @@ def remote_patron_lookup(
         if not data or "fields" not in data:
             return None
 
+        patrondata.complete = True
         fields: dict = data["fields"]
         patrondata.personal_name = fields.get("displayName")
         patron_type: str = fields["patronType"].get("key", "")
-
-        patrondata.external_type = patron_type
+        patrondata.library_identifier = patron_type
```

Comment:
Putting the `patron_type` into `library_identifier` lets us use the `library_identifier_restriction` settings on it.